src/planner/filter/zone_map_filter.cpp

```diff
@@ -0,0 +1,39 @@
+#include "duckdb/planner/filter/zone_map_filter.hpp"
+#include "duckdb/planner/filter/constant_filter.hpp"
+#include "duckdb/planner/table_filter.hpp"
+#include "duckdb/planner/expression.hpp"
+
+namespace duckdb {
+
+ZoneMapFilter::ZoneMapFilter() : TableFilter(TableFilterType::ZONE_MAP) {
```

Comment:
Let's rename the `ZoneMapFilter` to an `OptionalFilter` - indicating the true meaning of this type of filter, namely that the table scan is **allowed** to filter rows based on this criteria, **but not required**, as opposed to our other table filters that actually enforce that the rows must be removed for correctness.